ara_plumes/models.py

```diff
@@ -780,3 +762,61 @@ def background_subtract(
         clean_vid[i] = cv2.subtract(frame, background_img_np)
 
     return clean_vid
+
+
+def flatten_edge_points(mean_points: PlumePoints, vari_points: PlumePoints) -> Float2D:
+    """
+    Convert points on center and edge to distances from each other for a given frame.
+    NOTE: if no pair can be form, points are rejected. If more than one candidate
+    pair is found, the first pair is selected.
+    """
+    pairs = _create_radius_pairs(mean_points, vari_points)
+    rad_dist = []
+    for radius, center_points, edge_points in pairs:
+        dist = np.linalg.norm(center_points - edge_points)
+        rad_dist.append((radius, dist))
+    return np.array(rad_dist)
+
+
+def _create_radius_pairs(
+    mean_points: PlumePoints, vari_points: PlumePoints
+) -> List[tuple[float, Float1D, Float1D]]:
+    """
+    Create pairs of coordinates points based on same radius.
+    NOTE: In the event multiple pairs canddidates are identified, the first
```

Comment:
```suggestion
    NOTE: In the event multiple pairs candidates are identified, the first
```

Also, you could append something like "Curves are assumed to be true functions of radius with matching discrete support."  That may be clearer from a math perspective than "pairs candidates," which describes the function's implementation.